prac_05/word_occurrences.py

```diff
@@ -0,0 +1,37 @@
+"""
+CP1404/CP5632 Practical
+Word Occurrences
+Estimate: 30 minutes
+Actual:   27 minutes
+"""
+
```

Comment:
This part is almost perfect, only if you can add some more comments or docstring to clarify your process, it would be perfect !